src/shared/is_node.js

```diff
@@ -23,6 +23,7 @@ const isNodeJS =
   typeof process === "object" &&
   process + "" === "[object process]" &&
   !process.versions.nw &&
-  !(process.versions.electron && process.type && process.type !== "browser");
+  !(process.versions.electron && process.type && process.type !== "browser") &&
+  process.argv0 !== "brightsign";
```

Comment:
This is the only solution I found (or with navigator agent). I can't find a more official solution in their documentation. I will contact their support to ask for more information.

Reply:
> This is the only solution I found (or with navigator agent).

That's most unfortunate, since a solution depending entirely on the order of the arguments seems *way too brittle* in general and could thus very easily break.

> I can't find a more official solution in their documentation. I will contact their support to ask for more information.

Please do, since I'm sorry to say that I don't really think that we can accept this as-is.
The preferred solution would be that "BrightSign" essentially does what both NW.js and Electron do, by exposing its presence through e.g. a `process.versions.brightsign`-entry instead.

Reply:
I contact Brightsign to ask them to complete the information in `process.versions.brightsign`. In the meantime, they advised me to check for the presence of BSDeviceInfo object. Thank you.

Reply:
> I contact Brightsign to ask them to complete the information in `process.versions.brightsign`.

In that case, I think it'd be preferable that we wait until they provide a more "proper" of way detecting their environment.

> In the meantime, they advised me to check for the presence of BSDeviceInfo object.

To me that still feels a bit "hacky", especially if there's a change that something better will be provided.